docs/STVM_EP.md

```diff
@@ -32,10 +32,10 @@ pip3 install numpy decorator attrs
 Clone this repo using the `--recursive` flag to pull all associated dependencies
 
 
-Build TVM from the tvm_update folder:
+Build TVM from the tvm folder:
 
 ```
-cd onnxruntime/cmake/external/tvm_update/
+cd onnxruntime/cmake/external/tvm/
```

Comment:
Why users need to manually build TVM first? If that, it seems we don't need to clone TVM. We can treat TVM as CUDA SDK. We ask the user provide the install path of the software. 

In whatever case, we still need to track the software with cgmanifest.json.

Reply:
It comes from PR #10019. The first draft requires to manually build TVM. There is also an ongoing work with PR #10260 (renaming).